dspace-api/src/main/java/org/dspace/content/DSpaceObjectServiceImpl.java

```diff
@@ -239,7 +246,7 @@ public void addMetadata(Context context, T dso, MetadataField metadataField, Str
         for (int i = 0; i < values.size(); i++) {
 
             if (authorities != null && authorities.size() >= i) {
-                if (StringUtils.equals(authorities.get(i), "virtual")) {
+                if (StringUtils.startsWith(authorities.get(i), "virtual::")) {
```

Comment:
"virtual::" probably won't be changed but if anyone needs to change it, it requires at least 3 changes in this class. Should it be a Constant?

Reply:
This has been replaced with a constant now

Reply:
:+1: